subjects/java/projects/lets-travel/audit/README.md

```diff
@@ -0,0 +1,155 @@
+
+##### Evaluate the security and user-friendliness of the payment process.
+
+###### Is the payment process secure and accommodating of various payment methods?
+
+##### Test if Travelers can submit feedback on their travel experiences without issues.
+
+###### Can Travelers submit feedback easily?
+
+##### Verify if the feedback is visible to relevant parties such as Travel Managers and Admins for quality assurance.
+
+###### Is the feedback accessible for quality control by Travel Managers and Admins?
+
+##### Assess the ability of Travel Managers to create and manage travel listings effectively.
+
+###### Do Travel Managers have the ability to manage travel listings effectively?
+
+##### Verify if Travel Managers can view and interact with subscriber lists for their travels.
+
+###### Can Travel Managers interact with subscriber lists effectively?
+
+##### Verify if Travel Managers can access analytics about their travel listings and subscriber feedback.
+
+###### Do Travel Managers have access to analytics on their listings?
+
+##### Ensure that Traveler profiles are comprehensive, displaying past participations, feedback given, and reports made.
+
+###### Are Traveler profiles detailed and informative?
+
+### Ensure Authentication and Authorization Security
+
+##### Check the security and straightforwardness of the login process for all user roles.
+
+###### Is the login process secure and straightforward for all user roles?
+
+##### Confirm that role-based access controls are correctly enforced, preventing unauthorized actions across the system.
+
+###### Are role-based access controls properly enforced?
+
+##### Verify if data is transmitted securely using SSL/TLS encryption.
+
+###### Is data transmission secured with SSL/TLS encryption?
+
+##### Check if sensitive data and credentials are managed securely, adhering to best practices.
+
+###### Are sensitive data and credentials handled securely?
+
+##### Assess if the system handles high traffic volumes effectively without significant performance degradation.
+
+###### Can the system manage high traffic volumes without performance loss (you still can apply any action in the app in less than 5s)?
+
+##### Check if there is a fallback mechanism to ensure continuity of core functionalities in case of a service failure.
+
+###### Is there a fallback mechanism for service failures?
+
+##### Verify if the user interface is responsive across different devices and screen sizes.
+
+###### Is the UI responsive on various devices?
+
+##### Ensure the UI facilitates easy navigation and access to features for users of all roles.
+
+###### Does the UI support easy navigation for all user roles?
+
+##### Check if the system adheres to data protection regulations and privacy laws.
+
+###### Does the system comply with data protection
+
+##### Verify if development and deployment practices align with industry standards for security and code quality, such as `OWASP` and `clean code principles`.
```

Comment:
From [this](https://github.com/01-edu/public/pull/2434#discussion_r1521599420)

It could be nice to include some concrete things to include about the 2 points mentioned: checking everything mentioned in the website of OWASP, for example, could take quite some time and it would be a very long task to do for an audit in my opinion.

About `clean code principle`: that's a quite vague requirements since it is not really a standard and it's quite subjective what's "clean code" or not. I would add some examples of things to check.

Reply:
i removed both term 'clean code principle' and 'OWASP ' and i replace it by asking the auditor to check code readability and separation and also to check about SQL injection and XSS.